benchmarks/scale_1k_20k/test_1k_20k.py

```diff
@@ -0,0 +1,77 @@
+import ray
+import ray.autoscaler.sdk
+
+import json
+import os
+from time import sleep, perf_counter
+from tqdm import tqdm, trange
+
+TEST_NUM_NODES = 1000
+MAX_ACTORS_IN_CLUSTER = 20000
+
+
+def num_alive_nodes():
+    n = 0
+    for node in ray.nodes():
+        if node["Alive"]:
+            n += 1
+    return n
+
+
+def scale_to(target):
+    while num_alive_nodes() != target:
+        ray.autoscaler.sdk.request_resources(bundles=[{"node": 1}] * target)
+        print(f"Current # nodes: {num_alive_nodes()}, target: {target}")
+        print("Waiting ...")
+        sleep(5)
+
+
+def test_max_actors():
+    # TODO (Alex): Dynamically set this based on number of cores
+    cpus_per_actor = 0.2
+
+    @ray.remote(num_cpus=cpus_per_actor)
+    class Actor:
+        def foo(self):
+            pass
+
+    actors = [
+        Actor.remote()
+        for _ in trange(MAX_ACTORS_IN_CLUSTER, desc="Launching actors")
+    ]
```

Comment:
Can you also measure the time it takes to just execute lines 38-41, and record that as another metric?

Reply:
Sure

Reply:
```
Actor launch time: 88.6231253609999 (20000 actors)
Actor ready time: 79.9705737170002 (20000 actors)
Total time: 168.5936990780001 (20000 actors)
```